guardian/manage-frontend
# Turn off S+ switch to non serviceable countries

## What does this change?
Due to VAT compliance reasons we cannot support certain countries from having the Supporter Plus product.
As part of this PR BillingCountry is checked and compared with the non-serviceable countries list. This PR changes behaviour on MMA if the billingCountry is a non serviceable country:
- On Account Overview 'Change to monthly + extras' button is no longer visible
- On /reccuringsupport (accessed via the Mange recurring support button) 'Change amount' button is not visible. The user cannot update their amount but can change their payment method or cance
- If a user tries to access the switch journey via the /switch url they are redirected back to the account overview



## How to test
Create a Recurring supporter sub with a billing account as one of the non-servicable countries (OR if you have the code available you can add the billing country of an existing RC to the list temporarily) and log into MMA. Check that you can't see the switch button on account overview, change amount button on manage recurring contribution and that you can't access /switch


## How can we measure success?



## Have we considered potential risks?



## Images


Account Overview without switch button 
![image](https://github.com/guardian/manage-frontend/assets/115992455/1449b1ed-54ab-47de-9d17-6d4e325b26d8)

Manage Recurring support page without Change amount button
![image](https://github.com/guardian/manage-frontend/assets/115992455/70996d22-52b4-4689-a3e6-74968747ff9c)


## Accessibility
